examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -27,8 +27,8 @@
 import org.apache.beam.sdk.PipelineResult;
 import org.apache.beam.sdk.coders.AvroCoder;
 import org.apache.beam.sdk.coders.DefaultCoder;
-import org.apache.beam.sdk.io.BigQueryIO;
 import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.io.gcp.bigquery.BigQueryIO;
```

Comment:
Could we move BigQueryIO and other IOs to the gcp package, and keep implementation details in bigquery and datastore sub-packages?

In users' code, I like to have:
import org.apache.beam.sdk.io.gcp.BigQueryIO;
import org.apache.beam.sdk.io.gcp.DatastoreIO;